estate/models/estate_property_offer.py

```diff
@@ -31,3 +33,26 @@ def _inverse_date_deadline(self):
         for record in self:
             record.validity = (record.date_deadline - record.create_date.date()).days
 
+    def action_accept_offer(self):
+        for record in self:
+            #check if there are other offers already accepted
+            offers = record.property_id.offer_ids
+            sold = False
+            for offer in offers:
+                if offer.status == 'accepted': sold = True
```

Comment:
Statements on distinct lines from branching, in case further behavior needs to be added later inside block.